test/unit/source/load_vector_tile.test.ts

```diff
@@ -0,0 +1,107 @@
+import {test, expect, vi, beforeEach} from '../../util/vitest';
+import {loadVectorTile, DedupedRequest, resetRequestQueue} from '../../../src/source/load_vector_tile';
+import type {RequestedTileParameters} from 'src/source/worker_source';
+
+const createScheduler = () => ({add: () => {}} as any);
+const arrayBufDelay = 1500;
```

Comment:
⛏️ Not sure of the convention in this code, but I like naming constants like this all caps `ARRAY_BUF_DELAY`.